srcs/app/http/HttpStatusCodes.cpp

```diff
@@ -1,6 +1,10 @@
 #include <HttpStatusCodes.hpp>
 #include <utility>
 
+const char valid_http_methods[8][8] = {"GET", "HEAD", "POST",
+	"PUT", "DELETE", "CONNECT", "OPTIONS", "TRACE"};
+const uint16_t valid_return_status[] = {301, 302, 303, 307, 308};
+
```

Comment:
What do you think about making these variable names either uppercase or [google style](https://google.github.io/styleguide/cppguide.html#Constant_Names) to emphasize them?

Reply:
Done!